src/include/sof/audio/component.h

```diff
@@ -694,7 +694,7 @@ static inline struct comp_dev *comp_alloc(const struct comp_driver *drv,
 #define comp_get_drvdata(c) \
 	(c->priv_data)
 
-#if defined UNIT_TEST || defined __ZEPHYR__
+#if defined UNIT_TEST || defined __ZEPHYR__ || defined CONFIG_LIBRARY_STATIC
```

Comment:
we usually check for `CONFIG_*` values directly, without `defined()`, i.e. `|| CONFIG_LIBRARY_STATIC` - in multiple occurrences

Reply:
Thanks, it seems to work